core/model/src/main/java/it/unibz/inf/ontop/model/term/functionsymbol/db/impl/DBInFunctionSymbolImpl.java

```diff
@@ -0,0 +1,87 @@
+package it.unibz.inf.ontop.model.term.functionsymbol.db.impl;
+
+import com.google.common.collect.ImmutableList;
+import it.unibz.inf.ontop.iq.node.VariableNullability;
+import it.unibz.inf.ontop.model.term.*;
+import it.unibz.inf.ontop.model.term.functionsymbol.db.DBInFunctionSymbol;
+import it.unibz.inf.ontop.model.type.DBTermType;
+import it.unibz.inf.ontop.model.type.TermType;
+import it.unibz.inf.ontop.utils.ImmutableCollectors;
+
+import java.util.function.Function;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class DBInFunctionSymbolImpl extends DBBooleanFunctionSymbolImpl implements DBInFunctionSymbol {
+
+    protected DBInFunctionSymbolImpl(int arity, DBTermType rootDBTermType, DBTermType dbBooleanTermType) {
+        super("DB_IN_" + arity, IntStream.range(0, arity)
+                .mapToObj(i -> (TermType) rootDBTermType)
```

Comment:
No need for the type cast: use .<TermType>mapToObj instead.

Reply:
Done, thanks a lot!